### DISCUSSION TITLE
Port Range Docker

### DISCUSSION THREAD

**Discussion by patrickcorrigan** [pion/webrtc#1869] [Q&A] [answered]
Hi, I'm running pion in a docker container and I'm just wondering what ports I need to expose. Is there a port range?

**Comment by at-wat** [comment, marked as answer]
Hi @patrickcorrigan!
By default, pion/webrtc uses full range of ephemeral ports.
(typical default ranges of the OSes: https://en.wikipedia.org/wiki/Ephemeral_port#Range)
The range can be configured via SettingEngine:
https://github.com/pion/webrtc/blob/d1839c7652bd447b939e7f2f95ffff43410e0eac/settingengine.go#L21-L24
